src/components/Table/InventoryTable.jsx

```diff
@@ -1,20 +1,20 @@
-import React from 'react';
-import LendBtn from '../../../components/btn/LendBtn';
+import LendBtn from '../btn/LendBtn';
+import { useState } from 'react';
 import {
   TableContainer,
   TableRow,
   TableHeader,
   TableColumn,
   Divider,
-  ItemLabel,
-  ItemCount,
+  EquipmentLabel,
+  EquipmentCount,
   AvailableCount,
   UnavailableCount,
   TotalCount,
 } from './InventoryTable.styles';
 
 const InventoryTable = () => {
-  const [items, setItems] = React.useState([
+  const [Equipments, setEquipments] = useState([
```

Comment:
```suggestion
  const [equipments, setEquipments] = useState([
```

변수는 camel case로 작성해주세요